CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 Changelog
 ==========
+### 0.37.0
+- introduce Brisket.Logger
+
```

Comment:
Let's update the changelog in a separate PR as other things may have been merged while this was open.